ui/pages/confirmations/components/send/amount-recipient/amount-recipient.test.tsx

```diff
@@ -13,16 +13,6 @@ import { AmountRecipient } from './amount-recipient';
 
 const MOCK_ADDRESS = '0xdB055877e6c13b6A6B25aBcAA29B393777dD0a73';
 
-const mockHistory = {
-  goBack: jest.fn(),
-  push: jest.fn(),
-};
-
-jest.mock('react-router-dom', () => ({
```

Comment:
Unnecessary and breaks when certain react-router features are used in the component tree